pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -274,13 +274,13 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(env model.Env
 				direction:        http_conn.INGRESS,
 			}
 			if listenerOpts.tlsMultiplexed {
```

Comment:
This and `authnPolicy.RequireTLSMultiplexing` are pretty badly named - again, describing implementation and not the feature. Maybe `acceptIstioAndLegacy` or something like that would be better?

Reply:
Done.